### DISCUSSION TITLE
Does the reference architecture support multiple DB instance "types" simultaneously? (In other words, could the "Private Data Subnet" contain redundant PostgresQL instances and, say, redundant MongoDB instances?)

### DISCUSSION THREAD

**Discussion by marijakstrazdas** [gruntwork-io/knowledge-base#318] [Help] [answered]
Does the reference architecture support multiple DB instance "types" simultaneously? (In other words, could the "Private Data Subnet" contain redundant PostgresQL instances and, say, redundant MongoDB instances?)

**Comment by yorinasub17** [comment, marked as answer]
The Reference Architecture deploys an RDS cluster using our [rds service module](https://docs.gruntwork.io/reference/services/data-storage/amazon-rds). The RDS service module supports expanding the cluster with:

- Multi-AZ capability (one or two AWS managed hot standbys in alternative Availability Zones)
- Dedicated read replica instances, up to the RDS limits.

The initial deployment will include a single RDS cluster, but you can add additional calls to the `rds` module to deploy separate clusters into your environments after handoff.

We do not support deploying MongoDB in the initial deployment of the Reference Architecture, nor do we have a supported module for managing MongoDB (our existing module has been deprecated and unmaintained as of 2019).
